resources/lib/emby/core/connection_manager.py

```diff
@@ -516,7 +482,7 @@ def _get_connect_user(self, user_id, access_token):
     def _server_discovery(self):
         
         MULTI_GROUP = ("<broadcast>", 7359)
-        MESSAGE = "who is EmbyServer?"
+        MESSAGE = "who is JellyfinServer?"
```

Comment:
Are you sure this corresponds to what we currently have in server? I'm not sure we kept `Server` part of the question...

Reply:
Yes